tests/test_data_importer.py

```diff
@@ -211,6 +238,8 @@ def test_augmentation_mix(data_dir):
 
     run_import("corpus", dataset, prefix)
 
+    AUG_MAX_RATE = 0.35
+    AUG_MIN_RATE = 0.01
     data_dir.print_tree()
```

Comment:
We already cover the basic scenarios with `run_task` to test integration. These unit tests are lower level and they are supposed to test a variety of test cases. I don't think the coverage would improve if we switched it to the `run_task` here. 

It would be easy to switch but I think we should aim at doing lower-level unit testing on the code level rather than the task level as it's faster, more focused and it's easier to mock things.